app/lib/efile/nc/d400_calculator.rb

```diff
@@ -33,6 +35,23 @@ def calculate_line_10b
         amount_per_child * @direct_file_data.qualifying_children_under_age_ssn_count.to_i
       end
 
+      STANDARD_DEDUCTIONS = {
+        head_of_household: 19125,
+        married_filing_jointly: 25500,
+        married_filing_separately: 12750,
+        qualifying_widow: 25500,
+        single: 12750,
+      }.freeze
+      def calculate_line_11
+        STANDARD_DEDUCTIONS[@intake.filing_status]
+      end
+
+      def calculate_line_12a
+        # Add Lines 9, 10b, and 11
+        # line 9 DeductionsFromFAGI is blank
```

Comment:
[why is line 9 blank? I see some notes from the jira story and looks like it wasn't included in the story but not sure what you're looking at to see that it was not included.](https://codeforamerica.atlassian.net/browse/FYST-371?focusedCommentId=11580)

Reply:
in those jira comments it links to [this task](https://codeforamerica.atlassian.net/browse/FYST-470) which says "line 9 is blank"—that's what i was looking at